### PR TITLE
Added links to the popover

### PR DESCRIPTION
**Pull request by sammyshj**
Added `View History`, `New Message`, `Options`, `Help` links to the popover.

### CODE REVIEW

**Comment by sammyshj** [comment]
This PR is still not ready to be merged. The coverage is not updated and the tests have still not been added.

**Comment by sammyshj** [comment]
@smcgregor Does the changes look good? Then, I will add tests for the same and squash the commits into one.

File: privly.safariextension/scripts/popover.js [hunk 1]
```diff
@@ -43,6 +43,26 @@ function deactivateExtension() {
...
+    safari.application.activeBrowserWindow.openTab().url = safari.extension.baseURI + this.getAttribute("data-popover-path");
+  }
+}
+
 // Set the activation UI
 $("#deactivateExtension").click(deactivateExtension);
 $("#activateExtension").click(activateExtension);
+
+// Open new windows from the links
+var links = document.getElementsByTagName("a");
```
Comment (smcgregor) [inline privly.safariextension/scripts/popover.js]: Add a CSS class to the relevant elements and use `$(".the_css_class_here").click(openPopoverPage);`

**Comment by smcgregor** [comment]
One change (commented above). Make sure you comment a summary of changes in
the PR, otherwise it doesn't end up in my queue.

**Comment by sammyshj** [comment]
@smcgregor I have updated the PR.
The overall changes made in the PR:
* Popover height has been incresed.
* Popover links have been added for `View History`, `New Message`, `Options`, `Help`.
* The links of the popover has been mentioned in a new attribute, `data-popover-path`
* All the links are now contained under a class, `popover_data`
* On clicking any of the newly added popover links, the `openPopoverPage()` function is called which opens the required page in a new window.

**Comment by smcgregor** [comment]
Looks good :)

Go ahead and squash and I'll merge.

**Comment by sammyshj** [comment]
@smcgregor Squashed the commits into one.

**Pull request merged**